app/main/views/notifications.py

```diff
@@ -0,0 +1,41 @@
+# coding: utf-8
+from flask_login import current_user
+from flask import (
+    flash,
+    redirect,
+    render_template,
+    request,
+    url_for
+)
+from .. import main
+from ... import data_api_client
+
+
+@main.route('/notifications/user-research', methods=["GET", "POST"])
+def user_research_consent():
+    if current_user.is_anonymous():
```

Comment:
Can we not use `from flask_login import login_required` to achieve this behaviour out of the box?